js/src/carousel.js

```diff
@@ -114,8 +116,8 @@ class Carousel {
     this._config            = this._getConfig(config)
     this._element           = element
     this._indicatorsElement = this._element.querySelector(Selector.INDICATORS)
-    this._touchSupported    = 'ontouchstart' in document.documentElement || navigator.maxTouchPoints > 0
     this._pointerEvent      = Boolean(window.PointerEvent || window.MSPointerEvent)
+    this._touchSupported    = 'ontouchstart' in document.documentElement || this._pointerEvent
```

Comment:
the original check for `navigator.maxTouchPoints > 0` was, i'd say, more appropriate. as pointer events (in modern browsers) are supported/present even if there's just a mouse, so you end up with unnecessary code being executed...unless you're doing this on purpose to cover the scenario where an (external) touch-capable display is attached at a later point after the page has already loaded, which is a bit niche...

Reply:
i guess if you did revert back to the `navigator.maxTouchPoints` check, you'd then need to also fake it in the unit tests so that that code gets run

Reply:
The unit tests wasn't in success because when you swipe on a Imagine with IE it start a drag event 😬 I'll to check maxTouchPoints and see if everything is fine

Reply:
Ok I restored the use of `navigator.maxTouchPoints`, it should be fine now 👍 I'll make some tests on mobile devices